target/linux/ath79/dts/ar9344_tplink_tl-wdrxxxx.dtsi

```diff
@@ -78,7 +78,7 @@
 	flash@0 {
 		compatible = "jedec,spi-nor";
 		reg = <0>;
-		spi-max-frequency = <25000000>;
+		spi-max-frequency = <28125000>;
```

Comment:
Wouldn't putting 30 or 33 here be better, as we are sure to be above the threshold, and it's still not much higher ...